specs/protocol/derivation.md

```diff
@@ -368,15 +368,17 @@ where:
 A channel is encoded by applying a streaming compression algorithm to a list of batches:
 
 ```text
-rlp_batches = []
+encoded_batches = []
 for batch in batches:
-    rlp_batches.append(batch)
+    encoded_batches ++ batch.encode()
```

Comment:
Using `++` here to emphasize that there is no structure to `encoded_batches`, i.e. it is _not_ a `[][]byte`.